src/data/set/basic.lean

```diff
@@ -1064,6 +1070,9 @@ ext $ λ x, ⟨λ ⟨y, _, h⟩, h ▸ mem_singleton _,
 by simp only [eq_empty_iff_forall_not_mem]; exact
 ⟨λ H a ha, H _ ⟨_, ha, rfl⟩, λ H b ⟨_, ha, _⟩, H _ ha⟩
 
+lemma image_nonempty {s : set α} (f : α → β) (h : s.nonempty) : (f '' s).nonempty :=
```

Comment:
Don't we have `nonempty.image` somewhere already?

Reply:
Line 1126 say:
```lemma nonempty.image (f : α → β) {s : set α} : s.nonempty → (f '' s).nonempty```

Reply:
rofl I am a twit. These are lemmas from the perfectoid project, and I checked them all to see if they were there, and noticed that this now was, but then managed to PR it anyway :-) I suspect it appeared after the nonempty refactor. I'll remove it.